README.md

```diff
@@ -192,6 +192,41 @@ $ jsonnet -J vendor -S -e 'std.manifestYamlDoc((import "mixin.libsonnet").promet
 $ jsonnet -J vendor -m files/dashboards -e '(import "mixin.libsonnet").grafanaDashboards'
 ```
 
+### Adding custom annotations to alerts
```

Comment:
I think your intention was to demonstrate how to add annotations to **_specific_** alerts, no?

Reply:
Changed this line to be more generic: "Customising alert annotations".